Makefile

```diff
@@ -40,9 +40,9 @@ ARCH ?= $(shell uname -m | sed 's/x86_64/amd64/')
 # Image URL to use all building/pushing image targets
 REGISTRY ?= quay.io
 ORG ?= cloud-bulldozer
-# Get the current branch/tag name
+# Get the current branch name
 # In case this is the master branch, rename it to latest
-VERSION ?= $(shell git describe --tags 2>/dev/null || git rev-parse --abbrev-ref HEAD | sed 's/master/latest/g')
+VERSION ?= $(shell git rev-parse --abbrev-ref HEAD | sed 's/master/latest/g')
```

Comment:
I don't understand this change. The original code was able to set VERSION to something like v1.1 if you are in the the tag v.1.1 or failback to set VERSION to the current branch name (in case of master this is updated to latest).

You have drop detecting the tag name in your implementation, e.g.:

Output using the original implementation:
```shell
rsevilla@wonderland ~/labs/ripsaw ((v0.1)) $ git --version
git version 2.31.1
rsevilla@wonderland ~/labs/ripsaw ((v0.1)) $ git describe --tags 2>/dev/null || git rev-parse --abbrev-ref HEAD | sed 's/master/latest/g'
v0.1
rsevilla@wonderland ~/labs/ripsaw (master) $ git checkout e2e-testing
Switched to branch 'e2e-testing'
rsevilla@wonderland ~/labs/ripsaw (e2e-testing) $ git describe --tags 2>/dev/null || git rev-parse --abbrev-ref HEAD | sed 's/master/latest/g'
e2e-testing
```

Reply:
Yes indeed, I too can understand the bash involved. Let me rephrase my previous message; the issue was/is that the "original code" would resolve to a tag even on an untagged branch such as `master`.
```
$ git branch
* master
$ git describe --tags
v0.1-31-g2d3dc8d
```

This isn't the case on other forks, so it must be an issue on just mine. I'll revert the change.